frontend/src/scenes/session-recordings/filters/OrderingFilters.tsx

```diff
@@ -0,0 +1,34 @@
+import { useActions, useValues } from 'kea'
+import { LemonRadio } from 'lib/lemon-ui/LemonRadio'
+
+import { sessionRecordingsPlaylistLogic } from '../playlist/sessionRecordingsPlaylistLogic'
+
+export const OrderingFilters = (): JSX.Element => {
+    const { orderBy } = useValues(sessionRecordingsPlaylistLogic)
+    const { setOrderBy } = useActions(sessionRecordingsPlaylistLogic)
+
+    return (
+        <LemonRadio
+            value={orderBy}
+            onChange={setOrderBy}
+            options={[
+                {
+                    value: 'start_time',
+                    label: 'Latest',
```

Comment:
should we have data-attrs here so that autocapture can be turned into nice actions to track usage?